webauthn/registration/formats/tpm.py

```diff
@@ -234,9 +234,11 @@ def verify_tpm(
     # [TPMv2-EK-Profile] section 3.2.9.
     try:
         # Ignore mypy because we're casting to a known type
-        ext_subject_alt_name: SubjectAlternativeName = cert_extensions.get_extension_for_oid(
-            ExtensionOID.SUBJECT_ALTERNATIVE_NAME
-        ).value  # type: ignore[assignment]
+        ext_subject_alt_name: SubjectAlternativeName = (
+            cert_extensions.get_extension_for_oid(
+                ExtensionOID.SUBJECT_ALTERNATIVE_NAME
+            ).value
+        )  # type: ignore[assignment]
```

Comment:
Out of curiosity, are you making these changes or is Black making them?

Reply:
All the formatting changes are Black's doing. I see that the project doesn't have black/mypy/etc. set up on CI, so I guess some changes have snuck in without being properly formatted. I can submit a separate PR with formatting changes and CI setup for running black/mypy etc if you want?